net/ampr-ripd/Makefile

```diff
@@ -0,0 +1,181 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ampr-ripd
+PKG_VERSION:=2.4
+PKG_RELEASE:=1
+PKG_SOURCE_URL:=http://www.yo2loj.ro/hamprojects
+PKG_MD5SUM:=5429401ab6f8f793448bef11029d7ead
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tgz
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+PKG_LICENSE:=GPL-2.0-only
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/$(PKG_NAME)    
+	SECTION:=net
+	CATEGORY:=Network
+	SUBMENU:=Routing and Redirection
+	DEPENDS:=+kmod-ipip +ip-full
+	TITLE:=Routing daemon for the AMPR network
+	URL:=http://www.yo2loj.ro/hamprojects
+	MAINTAINER:=Dan Srebnick <k2ie@k2ie.net>
+endef
+
+define Package/$(PKG_NAME)/description
+	Routing daemon written in C similar to Hessu's rip44d including optional resending of RIPv2 broadcasts for router injection.
+endef
+
+CONFIGURE_VARS+= \
+	CC="$(TOOLCHAIN_DIR)/bin/$(TARGET_CC)"
+	COPT="$(TARGET_COPT)"
+
+define Package/$(PKG_NAME)/install
+	$(INSTALL_DIR) $(1)/usr/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/$(PKG_NAME) $(1)/usr/sbin
+	$(INSTALL_DIR) $(1)/etc
+	$(INSTALL_DIR) $(1)/etc/init.d
+	$(INSTALL_BIN) ./files/$(PKG_NAME)-init $(1)/etc/init.d/$(PKG_NAME)
+endef
+
+define Package/$(PKG_NAME)/postinst
```

Comment:
It looks like you are trying using something, which is not intented to do. This is no-go from my side. You should look to much more suitable solution

Reply:
To clarify, are you stating here that you don't approve of using the postinst script to configure the various network/firewall settings?  One of my major objectives in providing this package is to achieve ease of configuration.  If that is the case, I ought to probably withdraw the pull request and reconsider my overall approach.